diagnostics/TUTT/doc/TUTT.rst

```diff
@@ -0,0 +1,120 @@
+
+Version & Contact info
+----------------------
+
+.. '-' starts items in a bulleted list:
+   https://docutils.sourceforge.io/docs/user/rst/quickref.html#bullet-lists
+
+- Version/revision information: version 1.0 (7/2/2021)
+- PI: Zhuo Wang, zhuowang@illinois.edu, DAS UIUC)
+- Developer/point of contact (Zhuo Wang, Gan Zhang, Chuan-Chieh Chang, and Jiacheng Ye/Zhuo Wang, zhuowang@illinois.edu, DAS UIUC)
+
+.. Underline with '^'s to make a third-level heading.
+
+Open source copyright agreement
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The MDTF framework is distributed under the LGPLv3 license (see LICENSE.txt).
+
+Functionality
+-------------
+
+The current package consists of following functionalities:
+
+(1) Calculates geostrophic zonal winds (Ug) using 200-hPa geopotential height with a fixed Coriolis parameter at 15N.
+
+(2) Identifies positions of the circumglobal contour of the long-term seasonal-mean Ug. The value of Ug can be specified by the user, usually ranges from 1 to 2 m/s. The zonal-mean latitude of the circumglobal contour is chosen as the reference latitude.
+
+(3) The TUTT index is estimated from the area where the circumglobal contour of seasonal-mean Ug extends southward of the reference latitude.
+
+(4) In addition to TUTT area, the package also calculates the strengths and central locations of TUTTs.
+
+(5) The reference latitude, area, strength, central location of TUTT, as well as the positions of circumglobal Ug contour can be saved as .txt files. User can choose whether the aforementioned information is outputted or not.
+
+(**) cropping.py can be referenced if code is needed to either shift the grid of your data
+or to crop your data to a specified region
+
+As a module of the MDTF code package, all scripts of this package can be found under
+``mdtf/MDTF_$ver/diagnostics/TUTT
+
+Required programming language and libraries
+-------------------------------------------
+
+Python3 packages: "netCDF4", "skimage", "numpy", "scipy", "shapely.geometry", "cartopy"
+
+Required model input variables
+-------------------------------
+
+Time-varying 2-D geopotential height fields at 200 hPa (unit: gpm). 200-hPa geopotneitla height should be the monthly-mean field. Horizontal resolution of the geopotential height data can be deceided by the user. Note that the longitude index of the data must goes from west to east (0-360E).
+
+
+References
+----------
+
+.. _ref-Muñoz1:
+
+Wang, Z., Zhang, G., Dunkerton, T. J., & Jin, F. F. (2020). Summertime stationary waves integrate tropical and extratropical impacts on tropical cyclone activity. Proceedings of the National Academy of Sciences of the United States of America, 117(37), 22720-22726. https://doi.org/10.1073/pnas.2010547117
+
+Chuan-Chieh Chang and Zhuo's paper is under developing...
```

Comment:
Replace "under developing" with "in preparation", and provide a manuscript title if available